### PR TITLE
fix(test): Fixing flakey e2e onboarding test in cypress

### LINKED ISSUE #16446: Flakey Onboarding Test
The test 'Can onboard to configure later' in the onboarding cypress test file is not correct anymore. Going to go update so it no longer fails

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #16446 

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by zoesteinkamp** [comment]
> i reject this solution unless you can tell me why a timeout is more robust than looking for dom elements to be available in this situation

We can look for DOM elements, we are currently confirming the route is correct, but we can do this as well. But the underlying problem will not be resolved, as what is currently happening is a time out response, its saying hey i dont see this route you claim we should be on, guess we didnt make it "FAIL". When really it just needs more time to load up the route. Some routes are particularly more heavy and ask for way more info which can take awhile to retrieve. 

BUT, i also dont like this as a solution either. Its a solution presented in the cypress docs and the cypress forms. I think we could do something better. But i think this is an okay band aid.

File: ui/cypress/e2e/onboarding.test.ts [hunk 1]
```diff
@@ -5,6 +5,23 @@ interface TestUser {
...
+    cy.flush()
+
+    cy.fixture('user').then(u => {
+      user = u
+    })
+
+    cy.visit('/')
+  })
+  it('Can redirect to onboarding page', () => {
+    cy.get('[data-testid="init-step--head-main"]', {timeout: 5000})
```
Comment (121watts) [inline ui/cypress/e2e/onboarding.test.ts]:
```suggestion
    cy.getByTestID("init-step--head-main")
```
You also shouldn't need the timeout as Cypress will already wait 30 seconds to find any element.

File: ui/cypress/e2e/onboarding.test.ts [hunk 2]
```diff
@@ -5,6 +5,23 @@ interface TestUser {
...
+  beforeEach(() => {
+    cy.flush()
+
+    cy.fixture('user').then(u => {
+      user = u
+    })
+
+    cy.visit('/')
+  })
+  it('Can redirect to onboarding page', () => {
```
Comment (121watts) [inline ui/cypress/e2e/onboarding.test.ts]: Can we add some space for these to breathe?
```suggestion

  it('Can redirect to onboarding page', () => {
```

**Comment by 121watts** [comment]
also, to verify this changes flakiness you can wrap any test in a for loop and have it run `n` times.

example:

```
for (let n = 0; n < 50; n++ {
  it('does a thing, () => {
   ....
  })
}
```

File: ui/cypress/e2e/onboarding.test.ts [hunk 3]
```diff
@@ -111,11 +111,13 @@ describe('Onboarding', () => {
       //advance to Quick Start
       cy.getByTestID('button--quick-start').click()
 
+      cy.wait(20000)
```
[the remark this answers is not in the document]
Comment (zoesteinkamp) [reply ui/cypress/e2e/onboarding.test.ts]: This has been removed, it was for testing and snuck into one of the commits

File: ui/cypress/e2e/onboarding.test.ts (hunk 1 above)
Comment (zoesteinkamp) [reply ui/cypress/e2e/onboarding.test.ts]: got it

File: ui/cypress/e2e/onboarding.test.ts (hunk 2 above)
Comment (zoesteinkamp) [reply ui/cypress/e2e/onboarding.test.ts]: space added!

**Pull request merged**